admin/apple-actions/index/class-export.php

```diff
@@ -203,6 +203,18 @@ public function fetch_exporter() {
 		 */
 		$byline = apply_filters( 'apple_news_exporter_byline', $byline, $post->ID );
 
+		/**
+		 * Filters the slug of an article before it is sent to Apple News.
+		 *
+		 * The slug is used for the Slug component, if it is active.
+     *
+     * @todo Implement this in postmeta so there is a default value managed by the plugin.
```

Comment:
Tabs, not spaces (this was me, when I resolved the merge conflict):
```suggestion
		 *
		 * @todo Implement this in postmeta so there is a default value managed by the plugin.
```